apps/api/src/domain/models/shared/common-command-errors/UuidNotAvailableForUseError.ts

```diff
@@ -3,6 +3,6 @@ import { AggregateId } from '../../../types/AggregateId';
 
 export default class UuidAlreadyInUseError extends InternalError {
     constructor(invalidId: AggregateId) {
-        super(`The ID: ${invalidId} is not available for use`);
+        super(`The UUID: ${invalidId} is not available because it is already in use`);
```

Comment:
Don't reword this. If you'd like, rename the class `IdAlreadyInUseError`. The fact that it's a UUID is more of an implementation detail.

Reply:
Just wanting to understand better.  Are there are cases where the UUID is not available for other reasons than it is already in use?  This change came from trying to understand the error messages returned from cli commands.

Reply:
Yes. It's also possible that the ID was not generated with our ID generation system. We use a "reservation" pattern to ensure that all IDs are first generated with our system (and marked as available at that point) then marked as no longer available when used.

Reply:
I reverted this for now. At the end of the day, this isn't really a user-facing error (unless something goes wildly wrong) so I don't want to overthink it.